src/main/scala/org/monarchinitiative/dosdp/cli/Query.scala

```diff
@@ -35,23 +35,40 @@ object Query {
       targets <- determineTargets(config).mapError(e => DOSDPError("Failure to configure input or output", e))
       reasonerFactoryOpt <- reasonerFactoryOptZ
       ontologyOpt <- config.common.ontologyOpt
+      ont <- ZIO.fromOption(ontologyOpt).orElseFail(DOSDPError("Can't run query; no ontology provided."))
```

Comment:
Can't fail here so soon: there is an option to just print the query. This can be done without providing an ontology.

Reply:
So pass an `Option[Model]` later on instead of a `Model`.

Reply:
I think I have fixed this...can you take another look?

Reply:
Thanks—I made a few changes to avoid Options when a method can't do anything without the value. I like to deal with the optional status at the call site instead.